VOXELPLA PLUS BLACK.xml.fdm_material

```diff
@@ -0,0 +1,252 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura" version="1.3">
+  <metadata>
+    <name>
+      <brand>VOXELPLA</brand>
+      <material>PLA</material>
+      <color>BLACK</color>
+      <label>VOXELPLA PLUS BLACK</label>
+    </name>
+    <version>8</version>
+    <GUID>da7d7354-792f-4e80-ac3b-59df1de531fa</GUID>
+    <color_code>#000000</color_code>
+    <description>The most reliable PLA PLUS on the planet. Best printed with 200C on the extruder and 45-60C on the bed, minimal retraction is needed. Each spool is $16.99 on WWW.VOXELPLA.COM</description>
+    <adhesion_info>Print on glass, buildtak, PEI sheets, or any other buildplate material and apply little bit of glue stick.</adhesion_info>
+  </metadata>
+  <properties>
+    <density>1.24</density>
+    <diameter>1.75</diameter>
+  </properties>
+  <settings>
+    <setting key="print temperature">200</setting>
+    <setting key="heated bed temperature">60</setting>

```

Comment:
Most the printers that you included seem to have a heated bed temperature of 55.
If you change this value to 
```suggestion
    <setting key="heated bed temperature">55</setting>
```
Then you can remove all the mentions of heated bed temperature in the printers and you adhere to the DRY principle:
https://www.plutora.com/blog/understanding-the-dry-dont-repeat-yourself-principle